src/test/java/com/google/api/codegen/testdata/discoveries/go/go_storage.v1.json.baseline

```diff
@@ -18,16 +18,12 @@ import (
 
   "golang.org/x/net/context"
   "golang.org/x/oauth2/google"
-
   "google.golang.org/api/storage/v1"
 )
 
 func main() {
   ctx := context.Background()
 
-  // Use Application Default Credentials for authentication when running locally.
-  // For more information, see:
-  // https://developers.google.com/identity/protocols/application-default-credentials
```

Comment:
Huh, we're removing this link explaining ADC? I wonder whether we should move it to the end of comment "2". We can discuss separately from this PR and do uniformly across languages.